docs/intro.md

```diff
@@ -0,0 +1,99 @@
+# Learn the Basics
+
+This chapter introduces you to the basic conception of few shot learning, and the framework of
+MMFewShot, and provides links to detailed tutorials about MMFewShot.
+
+## What is Few Shot Learning
+
+### Problem definition
+Few-shot learning aims at generalizing to new tasks based on a limited number of samples using prior knowledge.
+The prior knowledge usually refers to a large scale training set which has many classes and samples,
+while the samples in new tasks are never seen in the training set.
+For example, in few shot image classification, a pre-trained model only can see
+five bird images (each class has one image and doesn't exist in the pretrained dataset) and
+predict the bird in query image belongs to which class.
+And in few shot detection, a detector needs to detect the new categories based on a few instances.
+
+In summary, few shot learning focus on two aspects:
+- how to embed prior knowledge into models (pre-train with large scale dataset)
+- how to transfer knowledge to adapt new tasks (learn on a few labeled samples).
+
+
+<div align=center>
+<img src="https://github.com/open-mmlab/mmfewshot/blob/main/resources/demo.png?raw=true" width=80%/>
+</div>
+
+### Terminologies in few-shot learning
+- Training set: every class in the training set has many samples, and it is big enough for learning a deep neural network.
+- Support set: a small set of labeled images and all the classes do not exist in training set.
+- Query set: unlabeled images to predict and share the same classes with support set.
+
+- N way K shot: the support set setting
```

Comment:
It means support images contain N classes and each class samples K images， so there will be N×K support images in each batch.